aep/general/0132/aep.md.j2

```diff
@@ -30,15 +30,15 @@ result **must** be a list of resources.
   - The response **should** usually include fully-populated resources unless
     there is a reason to return a partial response (see AEP-157).
 - The HTTP verb **must** be `GET`.
-- The URI **should** contain a single variable field corresponding to the
-  collection parent's name.
+- If the collection has a parent resource, The URI **should** contain a single
+  variable field corresponding to the collection parent's name.
```

Comment:
I know this PR doesn't introduce this language, but what is a "variable field"?

Reply:
I think it refers to the syntax in the URI - that said I agree this is vague. It is really just a field - I'll update the wording as such.